clients/fossid-webapp/src/main/kotlin/api/identification/markedAsIdentified/MarkedAsIdentifiedFile.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2020 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.fossid.api.identification.markedAsIdentified
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.fasterxml.jackson.annotation.JsonUnwrapped
+
+import org.ossreviewtoolkit.fossid.api.identification.common.Component
+import org.ossreviewtoolkit.fossid.api.summary.License
+import org.ossreviewtoolkit.fossid.api.summary.Summarizable
+import org.ossreviewtoolkit.fossid.api.summary.SummaryIdentifiedFile
+
+data class MarkedAsIdentifiedFile(
```

Comment:
Don't be abused if the code is looking the same, they are differences because both work on different `File` and `License` classes.
E.g. `IdentifiedFile` uses `org.ossreviewtoolkit.fossid.api.identification.identifiedFiles.File` and `org.ossreviewtoolkit.fossid.api.identification.identifiedFiles.License`.
`MarkedAsIdentifiedFile uses org.ossreviewtoolkit.fossid.api.identification.markedAsIdentified.File` and `org.ossreviewtoolkit.fossid.api.identification.markedAsIdentified.License`.

Here I duplicated the code on purpose during reverse engineering to isolate the different API calls.
When FossID releases their data model specs, I will be more than happy to rework it in ORT.
In the time being, I would feel more comfortable if we leave it as if.